apps/vendor-portal/src/components/notification-dropdown.vue

```diff
@@ -169,9 +177,23 @@ export default defineComponent({
       return "#87b563";
     };
 
+    const handleClick = (notifyType: string) => {
+      const low = notifyType.toLowerCase();
+
+      if (low.includes("import")) {
+        const page = props.items.find(
+          (page: { title: string }) => page.title === "Import"
```

Comment:
props.items is typeof IMenuItems[] - it's ok to use `page: IMenuItems` here to be consistent